code/modules/clothing/neck/_neck.dm

```diff
@@ -185,26 +185,91 @@
-	if(!user.can_hear())
-		diagnosis = "Fat load of good it does you though, since you can't hear"
-
-	to_chat(user, span_notice("You place [src] against [carbon_patient]'s [body_part]. [diagnosis]."))
+	//assess breathing
+	if(body_part == BODY_ZONE_CHEST)//Check if we can actually hear lung sounds
+		if(carbon_patient.stat == DEAD || (HAS_TRAIT(carbon_patient, TRAIT_FAKEDEATH)) || carbon_patient.failed_last_breath || carbon_patient.losebreath)//If pt is dead or otherwise not breathing
+			render_list += "<span class='danger ml-1'>[M.p_theyre(TRUE)] not breathing!</span>\n"
+		else if(lungs.damage > 10)//if breathing, check for lung damage
+			render_list += "<span class='danger ml-1'>You hear fluid in [M.p_their()] lungs!</span>\n"
+		else if(oxy_loss > 10)//if they have suffocation damage
+			render_list += "<span class='danger ml-1'>[M.p_theyre(TRUE)] breathing heavily!</span>\n"
+		else
+			render_list += "<span class='notice ml-1'>[M.p_theyre(TRUE)] breathing normally.</span>\n"//they're okay :D
+
+	//assess heart & blood level
+	if(body_part == BODY_ZONE_CHEST)//if we're listening to the chest
+		if(!heart.beating || carbon_patient.stat == DEAD)
+			render_list += "<span class='danger ml-1'>You don't hear a heartbeat!</span>\n"//they're dead or their heart isn't beating
+		else if(heart.damage > 10 || carbon_patient.blood_volume <= BLOOD_VOLUME_OKAY)
+			render_list += "<span class='danger ml-1'>You hear a weak heartbeat.</span>\n"//their heart is damaged, or they have critical blood
+		else
+			render_list += "<span class='notice ml-1'>You hear a healthy heartbeat.</span>\n"//they're okay :D
+	else if(body_part != BODY_ZONE_CHEST && body_part != BODY_ZONE_PRECISE_GROIN && body_part != BODY_ZONE_PRECISE_EYES && body_part != BODY_ZONE_PRECISE_MOUTH)//we're checking their pulse on an extremity
+		if(!heart.beating || carbon_patient.blood_volume <= BLOOD_VOLUME_OKAY || carbon_patient.stat == DEAD)
+			render_list += "<span class='danger ml-1'>You can't find a pulse!</span>\n"//they're dead, their heart isn't beating, or they have critical blood
+		else
+			if(heart.damage > 10)
+				heart_strength = "<span class='danger'>irregular</span>"//their heart is damaged
+			else
+				heart_strength = "<span class='notice'>regular</span>"//they're okay :D
+
+			if(carbon_patient.blood_volume <= BLOOD_VOLUME_SAFE && carbon_patient.blood_volume > BLOOD_VOLUME_OKAY)
+				pulse_pressure = "<span class='danger'>thready</span>"//low blood
+			else
+				pulse_pressure = "<span class='notice'>strong</span>"//they're okay :D
+
+			render_list += "<span class='notice ml-1'>[M.p_their(TRUE)] pulse is [pulse_pressure] and [heart_strength].</span>\n"
+
+	//assess abdominal organs
+	if(body_part == BODY_ZONE_PRECISE_GROIN)
+		if(liver.damage > 10 || appendix.damage > 10)//if there's damage to either their liver or appendix
```

Comment:
Aaaand same here, sanity check for liver and appendix

Reply:
Reshuffled their section a bit to make it work, but added!